libgetgood/monitor.go

```diff
@@ -52,6 +55,16 @@ func (monitor *Monitor) work() {
 	Logger.Debugf("Monitor stopped")
 }
 
+func (monitor *Monitor) logRequestsPerSecond() {
+
+	requestDiff := TotalRequestCount - monitor.requestsCounted
```

Comment:
Is there a reason we diff the time and the requests here?

Could we not just store the start time of the monitor and then do `TotalRequestCount / time.Since(monitor.startTime)`?